src/service/resources-impl.js

```diff
@@ -253,6 +257,11 @@ export class ResourcesImpl {
     });
 
     this.rebuildDomWhenReady_();
+
+    listen(this.win.document, 'scroll', e => this.scrolled_(e), {
+      capture: true,
+      passive: true,
+    });
```

Comment:
Should the handler be debounced?

Reply:
It's a bit risky to throttle or debounce, because both would drop the intermediate events for the last event. Eg, if I were to scroll two separate scrollers quickly, rate-limiting would mean I wouldn't receive the scroll event fir the first scroller.

I'm hoping the actual listener is fast enough that we don't have any issue.

Reply:
Probably rare in practice. You could have a custom debounce keyed to the event target if you really want to, but I don't think it's necessary for a quick patch.

Reply:
Done.